internal/gatewayapi/helpers.go

```diff
@@ -400,6 +406,10 @@ func irStringKey(gatewayNs, gatewayName string) string {
 	return fmt.Sprintf("%s/%s", gatewayNs, gatewayName)
 }
 
+func irMergedInfraPortName(listener *ListenerContext) string {
+	return fmt.Sprintf("%s/%s/%s", listener.Name, listener.gateway.Namespace, listener.gateway.Name)
```

Comment:
can we replace this with `return irHTTPListenerName(listener)` to improve code reuse ?

Reply:
sure